fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/postgresql/PostgreSqlAdapter.java

```diff
@@ -150,7 +150,22 @@ public void dropType(String schemaName, String typeName) {
 
     @Override
     public void createOrReplaceProcedure(String schemaName, String procedureName, Supplier<String> supplier) {
-        warnOnce(MessageKey.CREATE_PROC, "Create procedure not supported in PostgreSql");
+        final String objectName = DataDefinitionUtil.getQualifiedName(schemaName, procedureName);
+        logger.info("Create or replace procedure " + objectName);
+
+        // Build the create procedure DDL and apply it
+        final StringBuilder ddl = new StringBuilder();
+        ddl.append("CREATE OR REPLACE FUNCTION ");
```

Comment:
I do not think this should be in place of createOrReplaceProcedure.  There should be a new method for createOrReplaceFunction. Functions are a common db mechanism, I think this should remain as 153, and a new element for createOrReplaceFunction should be added.

Reply:
I would prefer to keep it simpler as it's now.
I chose to use postgresql function(or fake stored procedure) instead of procedure simply because postgresql doesn't have out parameter support at present, when postgresql support out parameter, then we can just change it to a real stored procedure.